lib/election.js

```diff
@@ -767,22 +767,33 @@ function _getManifestFromPeer(
             ledgerNode, peerId, eventHash, callback),
           callback);
       }
-      // type is 'RollCall', we already have the events votes from the peer
-      // in the database, so we just have to make sure that those listed in the
-      // manifest also exist in our database or else the manifest is bogus
-      // TODO: make sure manifest.items are in the database as events votes
       // TODO: must also ensure that manifest.items has >= twoThirds of
       //  electors.length... otherwise the roll-call is insufficient to get
       //  consensus!
       callback();
     }],
     validate: ['sync', (results, callback) => {
+      const manifest = results.manifest;
       if(type === 'Events') {
-        const manifest = results.manifest;
         return _validateEvents(
           ledgerNode, manifest.item, manifest.blockHeight, callback);
       }
-      callback();
+      // type === 'RollCall', we already have the events votes from the peer
+      // in the database, so we just have to make sure that those listed in the
+      // manifest also exist in our database or else the manifest is bogus
+      async.every(manifest.item, (voteHash, callback) =>
+        api._storage.votes.exists(ledgerNode.id, voteHash, callback),
```

Comment:
I guess that vote hashes can only get stored if what the hash was on was verified -- meaning it would have to be different data for a vote on voters (rollcall) vs. a vote on manifests. So it's probably ok to omit it. That's fine, let's omit, no change here.